src/screens/NestedInfoScreen.tsx

```diff
@@ -53,9 +68,14 @@ export const NestedInfoScreen = ({ navigation, route }: StackScreenProps<any>) =
   const [refreshing, setRefreshing] = useState(false);
   const name = route.params?.name;
   const rootRouteName = route.params?.rootRouteName;
+  const subQuery = route.params?.subQuery ?? undefined;
```

Comment:
we do not need to fallback to `undefined`. if there is no `subQuery` it's undefined. i will update that.